<?xml version="1.0"?>
<!-- Launches an UVC camera, the ball detector and its visualization  -->

<launch>
    <arg  name="config_path" default="$(find ball_detector)/launch/ball_detector_params.yaml"/>
    <!-- OP camera -->

    <!-- ball detector     -->
    <node pkg="ball_detector" type="ball_detector_node" name="ball_detector_node" args="" output="screen">
        <rosparam command="load" file="$(arg config_path)"/>  
	<!-- yaml path to save config -->
        <param name="yaml_path" type="string" value="$(arg config_path)"/>
        <remap from="/ball_detector_node/image_in" to="/image_raw" />
        <remap from="/ball_detector_node/cameraInfo_in" to="/camera_info" />
    </node>
</launch>

